### PR TITLE
w01d05

### PR DESCRIPTION
**Pull request by costajohnt**
comfort: 3
completeness: base 5, stretch 1

I spent too much time this weekend stuck on one piece of the problem .  In the future, if I'm stuck for more than a half hour, I'll look at the solution for a hint and try to reverse engineer faster.  I think I have the logic written out  pretty well for deciding who wins the game (in my notes), but I'm having trouble figuring out how to write it out in sublime make it work.  Instead of working on the stretch goals too much on Sunday, I chose to review some concepts I needed practice with and look at jquery tutorials.

### CODE REVIEW

**Comment by costajohnt** [comment]
comfort: 2
completeness: 5

I mean, I gave it my best shot.  I read through the information on OOP and closures several times, and also looked at additional resources.  I found some more exercises on closures and tried somewhat successful to work through them.  I researched analogies for OOP and I think I have a basic idea of its structure.  I'm definitely feeling like I'm behind on understanding the material and I just hope that's normal.

**Comment by costajohnt** [comment]
comfort:5
completeness:5

**Comment by tgaff** [comment]
@costajohnt if you get stuck try chatting or working together with other classmates

File: costajohnt/events_lab [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 0f9f69b4e746e52c8b24bcbf850139b0de86e090
```
Comment (tgaff) [inline costajohnt/events_lab]: This often indicates that you copied another github repository under your 23-modules repository. Unfortunately that won't work. If this is a lab that you were asked to fork, please remove it from here and put it in your `dev/` directory. Then push to the fork and make a pull-request on **that lab's repo**

If this is just a few files you can instead remove the hidden .git directory from `23-homework/jsnow/some_lab/.git` (warning don't remove `23-modules/.git`)  Then commit on this repo and re-push.  If that doesn’t make the files available to add in git you’ll also have to `git rm —cached some_lab`  Make sure you back up the files first.

**Pull request merged**